paper/paper.md

```diff
@@ -0,0 +1,164 @@
+te wai
+^[https://niwa.co.nz/natural-hazards/research-projects/mā-te-haumaru-ō-te-wai-increasing-flood-resilience-across-aotearoa]
+. To process LiDAR point cloud data from LINZ into a form ready to use for modelling the flow of water over the terrain
+surface we use GeoFabrics [@geofabrics]. The hydrologically-conditioned DEMs produced by GeoFabrics, along with
+additional data sources such as rainfall estimation from NIWA's High-Intensity Rainfall Design System and river network
+data from their River Environment Classification, are used as inputs to the flood modelling stage. The FReDT currently
+uses BG-Flood [@bg-flood] as underlying the hydrodynamic model for simulating flood events. BG-Flood was chosen because
+it aligns with Mā te haumaru ō te wai and has support from the NIWA authors. By using the methods developed for Mā te
+haumaru ō te wai, we intend to share scenarios created in the digital twin with NIWA, and ingest scenarios created
+externally into our tool.
+
+![Screenshot of a FReDT scenario, showing flooded areas with buildings highlighted in red if they are inundated.](Capture2024.png)
+
+# External data inputs
+
+The following list shows various data providers and datasets ingested by the FReDT:
+
+* LINZ
+    * LiDAR Digital Surface Models
+    * DEMs
+    * Building Outlines
+    * Coastlines
+    * Roads
+* NIWA
+    * River Environment Classification
+    * High-Intensity Rainfall Design System
+    * Tide
+* Takiwā
+    * Sea Level Rise
+* OSM
+    * Waterways
+* StatsNZ
+    * Region Geometries
+* Ministry for the Environment
+    * Sea Draining Catchments
+
+# Data outputs
+
+On the first scenario run for an area of interest, hydrologically conditioned DEMs are created from LiDAR data using the
+GeoFabrics package [@geofabrics]. These are reused for subsequent scenarios until the LiDAR data is updated, in which
+case they are regenerated. These hydrologically conditioned DEMs are generated using the same process as the NIWA-led
+national flood research program, Mā te haumaru ō te wai. These form part of the inputs for the BG-Flood model, but can
+also be downloaded through the web API to be used for further analysis.
+
+On every scenario run, we create multiple data outputs specific to the scenario. The primary data output is the flood
+model output, created using the shallow water hydrodynamic model BG-Flood [@bg-flood], which contains time-series
+geospatial rasters of inundation depths, water surface elevation, ground surface elevation, and water flow velocities in
+NetCDF format. In addition, we perform further analysis incorporating building footprint data to create a building flood
+status dataset that specifies whether each building is inundated past a threshold of 0.1m.
+
+# Statement of need
+
+Flooding, occurring approximately every eight months on average in Aotearoa New Zealand, stands as the most frequent,
```

Comment:
I think we should change this to `major floods` occurring approx every eight months

Reply:
Don't forget this one ;)